### PR TITLE
feat(genai): Updates to comments

### PR DESCRIPTION
**Pull request by CadillacBurgess1**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: generative_ai/batch_predict/gemini_batch_predict_bigquery.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ def batch_predict_gemini_createjob(output_uri: str) -> str:
     from vertexai.batch_prediction import BatchPredictionJob
 
     # TODO(developer): Update and un-comment below lines
+    # PROJECT_ID = "your-project-id
```
Comment (gericdong) [inline generative_ai/batch_predict/gemini_batch_predict_bigquery.py:31]: - Missing a quote

File: generative_ai/batch_predict/gemini_batch_predict_bigquery.py [hunk 2]
```diff
@@ -28,6 +28,7 @@ def batch_predict_gemini_createjob(output_uri: str) -> str:
     from vertexai.batch_prediction import BatchPredictionJob
 
     # TODO(developer): Update and un-comment below lines
```
Comment (gericdong) [inline generative_ai/batch_predict/gemini_batch_predict_bigquery.py:30]: Maybe remove 's' from 'lines'

File: generative_ai/batch_predict/gemini_batch_predict_gcs.py [hunk 3]
```diff
@@ -29,6 +29,7 @@ def batch_predict_gemini_createjob(output_uri: str) -> str:
     from vertexai.batch_prediction import BatchPredictionJob
 
     # TODO(developer): Update and un-comment below lines
+    # PROJECT_ID = "your-project-id
```
Comment (gericdong) [inline generative_ai/batch_predict/gemini_batch_predict_gcs.py:32]: Missing a quote

File: generative_ai/batch_predict/gemini_batch_predict_gcs.py [hunk 4]
```diff
@@ -29,6 +29,7 @@ def batch_predict_gemini_createjob(output_uri: str) -> str:
     from vertexai.batch_prediction import BatchPredictionJob
 
     # TODO(developer): Update and un-comment below lines
```
Comment (gericdong) [inline generative_ai/batch_predict/gemini_batch_predict_gcs.py:31]: Maybe remove 's' from 'lines'

File: generative_ai/batch_predict/gemini_batch_predict_gcs.py (hunk 3 above)
Comment (CadillacBurgess1) [reply generative_ai/batch_predict/gemini_batch_predict_gcs.py]: fixed.

File: generative_ai/batch_predict/gemini_batch_predict_bigquery.py (hunk 2 above)
Comment (CadillacBurgess1) [reply generative_ai/batch_predict/gemini_batch_predict_bigquery.py]: fixed.

File: generative_ai/batch_predict/gemini_batch_predict_gcs.py (hunk 4 above)
Comment (CadillacBurgess1) [reply generative_ai/batch_predict/gemini_batch_predict_gcs.py]: fixed.

**Review by gericdong — APPROVED** [review]

**Pull request merged**